galleries/examples/shapes_and_collections/hatchcolor_demo.py

```diff
@@ -0,0 +1,43 @@
+"""
+===========================
+Using hatchcolor in patches
+===========================
+
+This example shows how to use the `hatchcolor` parameter to set the color of
+the hatch. The `hatchcolor` parameter is available for `~.patches.Patch` and
```

Comment:
```suggestion
This example shows how to use the *hatchcolor* parameter to set the color of
the hatch. The *hatchcolor* parameter is available for `~.patches.Patch` and
```

This is failing the tests. Single ticks are references in Sphinx/ReStructuredText.